docs/migration_ha_0116.md

```diff
@@ -16,7 +16,7 @@ We've changed the default behavior of the `ignore_internal_state` attribute. It
 If you encounter issues with your current automations please set it to `False` again. We've analysed the current state
```

Comment:
We should delete this file. HA doesn't like external links.

Reply:
I have linked to this in the forums. It's easier to find than the knx breaking changes section of HAs blog (wich has no anchor links 🤦)

Reply:
Then create a new file for 0.117.0 instead of changing this one for which the beta has already been released?